app/src/androidTest/java/org/mozilla/fenix/ui/SettingsPrivacyTest.kt

```diff
@@ -133,6 +138,75 @@ class SettingsPrivacyTest {
         }
     }
 
+    @Test
+    fun saveLoginFromPromptTest() {
+        val saveLoginTest =
+                TestAssetHelper.getSaveLoginAsset(mockWebServer)
+
+            navigationToolbar {
+            }.enterURLAndEnterToBrowser(saveLoginTest.url) {
+                mDevice.waitNotNull(Until.findObjects(By.text("test@example.com")), TestAssetHelper.waitingTime)
+                val submitButton = mDevice.findObject(By.res("submit"))
+                submitButton.clickAndWait(Until.newWindow(), TestAssetHelper.waitingTime)
+                // Click save to save the login
+                mDevice.waitNotNull(Until.findObjects(By.text("Save")))
+                mDevice.findObject(By.text("Save")).click()
+            }.openHomeScreen {
+            }.openThreeDotMenu {
+            }.openSettings {
+                TestHelper.scrollToElementByText("Logins and passwords")
+            }.openLoginsAndPasswordSubMenu {
+                verifyDefaultView()
+                verifyDefaultValueSyncLogins()
+            }.openSavedLogins {
+            verifySavedLoginsView()
+            tapSetupLater()
+            // Verify that the login appears correctly
+            verifySavedLoginFromPrompt()
+        }
+    }
+
+    @Test
+    fun doNotSaveLoginFromPromptTest() {
+        val saveLoginTest = TestAssetHelper.getSaveLoginAsset(mockWebServer)
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(saveLoginTest.url) {
+            mDevice.waitNotNull(Until.findObjects(By.text("test@example.com")), TestAssetHelper.waitingTime)
+
+            val submitButton = mDevice.findObject(By.res("submit"))
+            submitButton.clickAndWait(Until.newWindow(), TestAssetHelper.waitingTime)
+            // Click save to save the login
+            mDevice.waitNotNull(Until.findObjects(By.text("Save")))
+            mDevice.findObject(By.text("Don’t save")).click()
+        }.openHomeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+            TestHelper.scrollToElementByText("Logins and passwords")
```

Comment:
I would move this into the openLoginsAndPasswordSubmenu function. This way anytime you expect to open the menu, the function will scroll there for you.